api/services/userService.js

```diff
@@ -0,0 +1,21 @@
+const bcrypt = require("bcrypt");
+const { userDao } = require("../models");
+
+const signUp = async (email, password, name, phoneNumber) => {
+  const saltRounds = 12;
+  const makeHash = async (password, saltRounds) => {
+    return await bcrypt.hash(password, saltRounds);
+  };
```

Comment:
굳이 함수로 또 만들 필요 없습니다. 아래 코드에서 바로 호출해 주세요.
```suggestion
  };
```